Automattic/wp-calypso
# A4A: Update tooltip for plugin updates that are host managed

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/505

## Proposed Changes

* Reverts https://github.com/Automattic/wp-calypso/pull/87257
* Uses the same logic to update the tooltip text when updates with managed plugins are available

<img width="497" alt="Screenshot 2024-07-23 at 15 06 28" src="https://github.com/user-attachments/assets/7b5b5162-50dc-4567-a779-915237171b0a">

## Testing Instructions



* Open the live link, login with demo account, and go to the sites dashboard
* Check that sites with updates available display either `Plugin updates are available` or `Some plugins are managed by the host and cannot be autoupdated`, depending on whether they have updates to managed plugins or not.
* Also ensure that sites listed in the "Needs Attention" view clearly indicate why they need attention.

## Pre-merge Checklist
